app/views/doc/_ext_courses.erb

```diff
@@ -8,6 +8,12 @@
           By Ferdinando Santacroce, Aske Olsson, Rasmus Voss and Jakub Narębski
         </p>
       </li>
+      <li>
```

Comment:
i think it would make sense to add this to a right column, WDYT?

Reply:
Yes, I’m happy to do that.